File: 3747-maximum-difference-between-adjacent-elements-in-a-circular-array/3747-maximum-difference-between-adjacent-elements-in-a-circular-array.cpp
class Solution {
public:
    int maxAdjacentDistance(vector<int>& nums) {
        int ans = abs(nums[0]-nums[1]);
        for(int i = 1; i< nums.size()-1; i++)
            {
                ans = max(ans,abs(nums[i]-nums[i+1]));
            }
        ans = max(ans, abs(nums[nums.size()-1]-nums[0]));
        return ans;
        
    }
};

File: 3747-maximum-difference-between-adjacent-elements-in-a-circular-array/3747-maximum-difference-between-adjacent-elements-in-a-circular-array.cs
public class Solution {
    public int MaxAdjacentDistance(int[] nums) {
        int ans = Math.Abs(nums[0]-nums[1]);
        for(int i = 1; i< nums.Length-1; i++)
            {
                ans = int.Max(ans,Math.Abs(nums[i]-nums[i+1]));
            }
        ans = int.Max(ans, Math.Abs(nums[nums.Length-1]-nums[0]));
        return ans;
        
    }
}